runtime/doc/lua.txt

```diff
@@ -692,6 +692,27 @@ identical identifiers, highlighting both as |hl-WarningMsg|: >
     ((binary_expression left: (identifier) @WarningMsg.left right: (identifier) @WarningMsg.right)
      (eq? @WarningMsg.left @WarningMsg.right))
 
+------------------------------------------------------------------------------
+VIM.HIGHLIGHT                  				*lua-highlight*
```

Comment:
I think maybe this should be lower case actually, just vim.highlight

Reply:
The other sections here are upper case as well (`VIM.TREESITTER` and `VIM.REGEX`), so I copied that. Can change it, though?

Reply:
Ah, ok. it's fine then (can always change later)